### DISCUSSION TITLE
Mastodon not working with storj

### DISCUSSION THREAD

**Discussion by Bobby99as** [mastodon/mastodon#18858] [Q&A]
Hello I'm trying to use storj as my S3 provider but it seems to not be working with Mastodon it works with everything else besides Mastodon it's S3 compatible

**Comment by Bobby99as** [comment]
Any help with this?

**Comment by Dark-Sheepa** [comment]
Hi, I see that nobody here has solved a similar problem, huh? I'm installing Mastodon on our server and I REALLY WANT to use StorJ to store data, but I can't establish a connection. Has anyone solved this problem? That he would share the result. Please...

**Comment by Dark-Sheepa** [comment]
I don't think I'll get any advice here. But still, if there is someone who will solve the same problem as me, I leave below a link to the discussion where we solve it. After all, why not use better and cheaper storage for data when I don't have to. See you.

https://forum.storj.io/t/adding-storj-to-mastodon/16753/
